### ISSUE TITLE
Error while reading ddccontrol home directory.: Resource temporarily unavailable

### ISSUE THREAD

**Issue by itJunky** [ddccontrol/ddccontrol#19]
What does it mean? Where ddccontrol home directory?
I run it by 'sudo gddccontrol' in ubuntu 17

**Comment by larstobi** [comment]
Could you please post the entire error message?

**Comment by itJunky** [comment]
`$ LC_ALL=C sudo gddccontrol 
[sudo] password for alp: 
Error while reading ddccontrol home directory.: Resource temporarily unavailable
`

**Comment by Ashark** [comment]
I got the same error. How to fix it?

**Comment by kravemir** [comment]
Did you install `ddccontrol-db`?

**Comment by Ashark** [comment]
Yes, I am using ArchLinux. I have ddccontrol (https://aur.archlinux.org/packages/ddccontrol/) package installed, and it has ddccontrol-db-git (https://aur.archlinux.org/packages/ddccontrol-db-git/) as a dependency.

**Comment by DanishMule** [comment]
Same issue here. pop_os (ubuntu) 3 monitor setup
:~$ sudo gddccontrol
Invalid response, corrupted data - xor is 0x2e, length 0x23
0000: 6e a3 e3 00 60 35 20 30 38 20 30 42 29 20 31 36 | n...`5 08 0B) 16
0010: 20 31 38 20 31 41 20 35 32 20 36 30 28 30 31 20 |  18 1A 52 60(01 
0020: 20 20 20 20 20 20 20 20 20 20 20 20 20 20 20 20 |                 
0030: 30 33 20 31 31 0d 20 35 32 20 36 30 28 30 31 20 | 03 11. 52 60(01 
0040: ff ff ff                                        | ...             
Invalid response, first byte is 0xff, should be 0x6e
0000: ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff ff | ................
0010: 20 31 38 20 31 41 20 35 32 20 36 30 28 30 31 20 |  18 1A 52 60(01 
0020: 30 33 20 31 31 0d 20 35 32 20 36 30 28 30 31 20 | 03 11. 52 60(01 
0030: ff ff ff ff ff ff 00 36 74 30 00 01 00 00 00 18 | .......6t0......
0040: 0f 01 03                                        | ...             
I/O warning : failed to load external entity "/usr/share/ddccontrol-db/monitor/AOC2775.xml"
Document not parsed successfully.
Error while reading ddccontrol home directory.: Resource temporarily unavailable
Invalid response, first byte is 0x00, should be 0x6e
0000: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | ................
0010: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | ................
0020: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | ................
0030: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | ................
0040: 00 00 00                                        | ...             
Invalid response, first byte is 0x00, should be 0x6e
0000: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | ................
0010: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | ................
0020: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | ................
0030: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | ................
0040: 00 00 00                                        | ...             
Invalid response, first byte is 0x00, should be 0x6e
0000: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | ................
0010: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | ................
0020: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | ................
0030: 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 | ................
0040: 00 00 00                                        | ...             
I/O warning : failed to load external entity "/usr/share/ddccontrol-db/monitor/MVP2200.xml"
Document not parsed successfully.
Error while reading ddccontrol home directory.: Resource temporarily unavailable

(gddccontrol:241459): Gtk-WARNING **: 19:47:00.407: gtk_widget_size_allocate(): attempt to allocate widget with width -1 and height 333

(gddccontrol:241459): Gtk-WARNING **: 19:47:00.407: gtk_widget_size_allocate(): attempt to allocate widget with width -1 and height 333
I/O warning : failed to load external entity "/usr/share/ddccontrol-db/monitor/ENC1918.xml"
Document not parsed successfully.
Error while reading ddccontrol home directory.: Resource temporarily unavailable

**Comment by DanishMule** [comment]
And when you try to send in monitor details as requested you get this returned:

Your mail to 'ddccontrol-users' with the subject

    ddccontrol -p -c -d

Is being held until the list moderator can review it for approval.

The reason it is being held:

    Post by non-member to a members-only list

Either the message will get posted to the list, or you will receive
notification of the moderator's decision.  If you would like to cancel
this posting, please visit the following URL:
